packages/cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@zowe/cli",
   "version": "8.0.0-next.202407232256",
-  "zoweVersion": "V3",
+  "zoweVersion": "v3.0.0",
```

Comment:
Would we want this to have a prerelease on it prior to 3.0.0?

Reply:
Having a prerelease suffix like `v3.0.0-prerelease` for now might be good :yum: When V3 goes live then running the workflow would set the version back to `v3.0.0` without the suffix.

Reply:
@ATorrise Should this comment be addressed?

Reply:
@t1m0thyj yes whoops ty